service_configuration_lib/spark_config.py

```diff
@@ -41,7 +41,7 @@ class ExecutorResourceConfig(NamedTuple):
 
 
 # Multi-step release - adjust if requested memory <= this threshold (memory in GB)
-ADJUST_EXECUTOR_MEM_CPU_RATIO_THRESH = 7
+ADJUST_EXECUTOR_MEM_CPU_RATIO_THRESH = 14
```

Comment:
i'd recommend moving this to srv-configs so that we don't need a version bump here and in paasta to update this value (and tbh, maybe the rest of the hardcoded values here to save us some work in the future)

Reply:
we can always mock things out for tests - and in any case, we're already using srv-configs in this file: https://github.com/Yelp/service_configuration_lib/blob/4ee2fad539ff8c17a0e8f43f72a9dfdc72b34019/service_configuration_lib/spark_config.py#L28

Reply:
Thanks! Will do it with the next release.